src/components/PlusMinusField/PlusMinusField.tsx

```diff
@@ -0,0 +1,117 @@
+import { StyleSheet, Text, TextStyle, View, TextInput } from 'react-native';
+import React, { useEffect } from 'react';
+
+import { useTheme } from '@hooks/useTheme';
+import { IconButtonV2 } from '@components/index';
+interface PlusMinusFieldProps {
+  labelStyle?: TextStyle | TextStyle[];
+  label: string;
+  value: number;
+  displayValue?: number | string;
+  min?: number;
+  max?: number;
+  valueChange?: number;
+  method: (value: number) => void;
+  onFocus?: (event: Event) => void;
```

Comment:
Is this being used anywhere?

Reply:
Originally I tried something that didn't work and I forgot about this.